api/src/main/java/org/eclipse/microprofile/config/ConfigValue.java

```diff
@@ -43,10 +43,17 @@
     /**
      * The value of the property lookup.
```

Comment:
It is better to say:
The value of the property lookup with transformation (expansion, etc).

It will be much clearer.